.github/workflows/ci_pr.yml

```diff
@@ -0,0 +1,99 @@
+name: CI
+
+# trigger
+on: [pull_request]
+
+jobs:
+  notebooks1:
```

Comment:
Maybe add some comment about the scope of each job and where new things should (not) be added?